File: server/models/Stat.js
const mongoose = require('mongoose');

const statSchema = new mongoose.Schema({
  label: { type: String, required: true },
  value: { type: String, required: true }, // e.g. "150+", "99.9%"
  suffix: { type: String, default: '' },
  description: { type: String, default: '' },
  icon: { type: String, default: 'Award' },
  order: { type: Number, default: 0 }
}, { timestamps: true });

module.exports = mongoose.model('Stat', statSchema);

File: server-java/src/main/java/com/denesens/model/Stat.java
package com.denesens.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.mongodb.core.mapping.Document;

import java.time.Instant;

@Data
@Document(collection = "stats")
public class Stat {
    @Id
    @JsonProperty("_id")
    private String id;
    
    private String label;
    private String value;
    private String suffix = "";
    private String description = "";
    private String icon = "Award";
    private Integer order = 0;

    @CreatedDate
    private Instant createdAt;

    @LastModifiedDate
    private Instant updatedAt;
}
